### PR TITLE
Scrut Fixes

### PR DESCRIPTION
**Pull request by Rosiak**

### CODE REVIEW

**Comment by f0o** [comment]
Do the snmp functions all return false or could they also return empty strings or 0 or NULL?

**Comment by Rosiak** [comment]
hmmmmmmmmmmmmm

**Comment by laf** [comment]
Is this still a valid pr @Rosiak ?

File: includes/discovery/sensors/current/apc.inc.php [hunk 1]
```diff
@@ -8,11 +8,11 @@
         $oids = snmp_walk($device, 'rPDULoadPhaseConfigIndex', '-OsqnU', 'PowerNet-MIB');
     }
 
-    if ($oids) {
+    if ($oids !== false) {
         d_echo($oids."\n");
 
         $oids = trim($oids);
```
Comment (murrant) [inline includes/discovery/sensors/current/apc.inc.php]: WIll trim(false) cause any issues?

File: includes/discovery/sensors/current/apc.inc.php [hunk 2]
```diff
@@ -171,12 +171,12 @@
 
     // ATS
     $oids = snmp_walk($device, 'atsConfigPhaseTableIndex', '-OsqnU', 'PowerNet-MIB');
-    if ($oids) {
+    if ($oids !== false) {
         $type = 'apc';
         d_echo($oids."\n");
 
         $oids = trim($oids);
```
Comment (murrant) [inline includes/discovery/sensors/current/apc.inc.php]: I'm pretty sure trim(false) == "".  Make sure you move the trims.

**Comment by Rosiak** [comment]
This was just meant as a fix for scrut issues, but I'm unsure if this is still valid due to the questions raised by @f0o .

**Pull request closed without merging**